RSSReader/View/FeedViewController.m

```diff
@@ -0,0 +1,87 @@
+//
+//  FeedViewController.m
+//  RSSReader
+//
+//  Created by Владислав Станкевич on 17.11.20.
+//
+
+#import "FeedViewController.h"
+#import "FeedCell.h"
+
+@interface FeedViewController ()
+
+@end
+
+@implementation FeedViewController
+
+- (void)viewDidLoad {
+    self.navigationController.navigationBar.hidden = YES;
+    [super viewDidLoad];
+    [self setupPresenter];
+    [self setupTableView];
+    [self setupConstraints];
+}
+
+- (void)setupPresenter {
+    NSMutableArray *feedItemArray = [[NSMutableArray alloc] init];
+    self.feedItemArray = feedItemArray;
+    [feedItemArray release];
+    FeedPresenter *presenter = [[FeedPresenter alloc] initWithArray:self.feedItemArray];
+    self.presenter = presenter;
+    [presenter release];
+    [self.presenter loadNewsWiithCompletion:^(BOOL success) {
+        if (success) {
+            [self.tableView reloadData];
+        }
+    }];
+}
+
+- (void)setupTableView {
+    UITableView *tableView = [[UITableView alloc] init];
+    self.tableView = tableView;
+    self.tableView.translatesAutoresizingMaskIntoConstraints = NO;
+    self.tableView.dataSource = self;
+    self.tableView.delegate = self;
+    [self.tableView registerClass:FeedCell.class forCellReuseIdentifier:@"cellId"];
+    [self.view addSubview:self.tableView];
+    [tableView release];
+}
+
+- (void)setupConstraints {
+    [NSLayoutConstraint activateConstraints:@[
+        [self.tableView.topAnchor constraintEqualToAnchor:self.view.topAnchor],
+        [self.tableView.leadingAnchor constraintEqualToAnchor:self.view.leadingAnchor],
+        [self.tableView.trailingAnchor constraintEqualToAnchor:self.view.trailingAnchor],
+        [self.tableView.bottomAnchor constraintEqualToAnchor:self.view.bottomAnchor],
+    ]];
+}
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    FeedCell *cell = [tableView dequeueReusableCellWithIdentifier:@"cellId"];
+    if (self.feedItemArray.count != 0) {
+        cell.feedItem = self.feedItemArray[indexPath.row];
```

Comment:
You should not expose internal properties. A better way to do it would be:

[cell configureWithItem: self.feedItemArray[indexPath.row]];